client/src/components/ratingsReviews/ratingProgress.css

```diff
@@ -0,0 +1,19 @@
+meter {
+  display: block;
+  --background: #CCCCCC;
+  --optimum: rgb(36, 155, 36);
+}
+
+meter::-webkit-meter-bar {
+  background: var(--background);
```

Comment:
What does `var(--background)` do?

Reply:
var() is a CSS function used to reference a custom property in the stylesheet which has (--) dashes in front of it. Useful when you have conditional rendering or you are referencing the same background/ custom property multiple times in the file. 

In this case, it was referencing the background from line 3, however, I've decided to remove var for now since I can work without it, but might use it if I need to refactor meter